### DISCUSSION TITLE
v1.7.0

### DISCUSSION THREAD

**Discussion by maxileith** [maxileith/homebridge-appletv-enhanced#619] [Announcements]
## Changed

* dependency updates
* improved logging

## Removed

* The option to expose as a Set-Top Box has been removed

---

## What's Changed
* Update homebridge/homebridge Docker tag to v20240903 by @renovate in https://github.com/maxileith/homebridge-appletv-enhanced/pull/594
* npm: Update dependency node-22 to v22.8.0 by @renovate in https://github.com/maxileith/homebridge-appletv-enhanced/pull/593
* npm: bump axios from 1.7.5 to 1.7.7 by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/601
* npm: bump @types/node from 22.5.1 to 22.5.3 by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/602
* pip: bump yarl from 1.9.4 to 1.9.8 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/599
* pip (ici): bump cryptography from 43.0.0 to 43.0.1 in /.github/issue_completeness_check by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/597
* pip: bump cryptography from 43.0.0 to 43.0.1 in /python_requirements/default in the crypt group by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/598
* npm: bump the typescript-eslint group with 2 updates by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/600
* pip: bump pyatv from 0.15.0 to 0.15.1 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/605
* pip: bump cffi from 1.17.0 to 1.17.1 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/606
* pip: bump yarl from 1.9.8 to 1.9.11 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/604
* pip (ici): bump cffi from 1.17.0 to 1.17.1 in /.github/issue_completeness_check by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/608
* npm: bump @types/node from 22.5.3 to 22.5.4 by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/607
* pip: bump the pydantic group across 1 directory with 2 updates by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/603
* npm: bump typescript from 5.5.4 to 5.6.2 by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/616
* npm: bump the typescript-eslint group with 2 updates by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/610
* pip: bump the pydantic group in /python_requirements/default with 2 updates by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/609
* pip: bump zeroconf from 0.133.0 to 0.134.0 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/611
* npm: bump homebridge-config-ui-x from 4.57.1 to 4.58.0 by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/612
* npm: bump the eslint group with 2 updates by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/617
* pip: bump yarl from 1.9.11 to 1.11.1 in /python_requirements/default by @dependabot in https://github.com/maxileith/homebridge-appletv-enhanced/pull/613
* v1.7.0 by @maxileith in https://github.com/maxileith/homebridge-appletv-enhanced/pull/618

**Full Changelog**: https://github.com/maxileith/homebridge-appletv-enhanced/compare/v1.6.1...v1.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/maxileith/homebridge-appletv-enhanced/releases/tag/v1.7.0'>v1.7.0</a>.</em>

**Comment by winstonstwn** [comment]
Hi, may I know why is the option to expose as Set-top Box removed? What will happen after update if I had mine exposed as stb before?

**Reply by maxileith** [reply]
It will be exposed as an Apple TV. Nothing should break on your side besides the icon in the home app.
